plugins/cli_rgb.py

```diff
@@ -30,35 +30,40 @@
 """
 
 from __future__ import division
+
 from collections import OrderedDict
 from concurrent.futures import CancelledError
 import logging
 from odemis import dataio, model
 from odemis.acq import stream, drift, acqmng
 from odemis.acq.stream import UNDEFINED_ROI
+from odemis.gui.conf import util
 import odemis.gui
 from odemis.gui.conf import get_acqui_conf
+from odemis.gui.plugin import Plugin, AcquisitionDialog
 import os.path
 import time
 import wx
-from odemis.gui.plugin import Plugin, AcquisitionDialog
 
 
 class RGBCLIntensity(Plugin):
     name = "RGB CL-intensity"
-    __version__ = "1.1"
-    __author__ = "Toon Coenen"
+    __version__ = "1.2"
+    __author__ = "Toon Coenen & Éric Piel"
```

Comment:
Place holder to ask a question via Slack.

Reply:
Indeed, this is incorrect, on Python2. Fixed.